services/mcp-memory/.env.example

```
@@ -0,0 +1,14 @@
+# MCP memory resource server (G6.M1). Copy to .env for local runs.
+
+IBEX_ENV=development
+IBEX_MCP_TRANSPORT=streamable_http
+IBEX_MCP_ALLOW_STDIO=false
+# Local default is loopback. Containers must set IBEX_MCP_HOST=0.0.0.0 explicitly.
+IBEX_MCP_HOST=127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IBEX_MCP_HOST key should go before the IBEX_MCP_TRANSPORT key

(UnorderedKey)